LibraryManager.sln

```diff
@@ -57,8 +57,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ShimGenerator", "ShimGenera
 	EndProjectSection
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AppHostRestorerProject", "AppHostRestorerProject", "{E9F67355-871B-4104-B887-0BC6D0A04B65}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AppHostRestorerProject", "tools\ShimGenerator\AppHostRestorerProject\AppHostRestorerProject.csproj", "{CE6EDDD4-AFBD-4FCF-89FC-30A0CD034642}"
+	ProjectSection(SolutionItems) = preProject
+		tools\ShimGenerator\AppHostRestorerProject\AppHostRestorerProject.csproj = tools\ShimGenerator\AppHostRestorerProject\AppHostRestorerProject.csproj
+	EndProjectSection
```

Comment:
This looks like a solution file, not a project in the solution?

Reply:
That is correct and is what I intended. 
This file is used by the msbuild magic to generate the shims and fails to restore when the solution restore runs.